src/main/java/gov/nih/nci/evs/api/service/MappingLoaderServiceImpl.java

```diff
@@ -77,6 +77,7 @@ public List<ConceptMap> buildMaps(final String mappingData, final String[] metad
             Arrays.copyOfRange(mappingDataList, 1, mappingDataList.length)) {
           final String[] conceptSplit = conceptMap.split("\t");
           final ConceptMap conceptToAdd = new ConceptMap();
+          conceptToAdd.setMapsetCode(metadata[0]);
```

Comment:
There's a lot of repeated code here. Since we are adding the same items to the ConceptMap, we should create a helper method to call instead of using duplicated code across multiple methods. This way we only have to update it in one spot instead of 3+

Reply:
moved repeated code to the top of the block outside of the if/else